src/MISC/fix_deposit.cpp

```diff
@@ -766,6 +776,13 @@ void FixDeposit::options(int narg, char **arg)
       vzlo = force->numeric(FLERR,arg[iarg+1]);
       vzhi = force->numeric(FLERR,arg[iarg+2]);
       iarg += 3;
+    } else if (strcmp(arg[iarg],"orientation") == 0) {
```

Comment:
error check for all components = 0.0, else the norm will blow up
keyword needs to be documented for 2d (and error checked),
or else disallowed for 2d if not supported.